lib/internal/test_runner/test.js

```diff
@@ -525,24 +524,25 @@ class Test extends AsyncResource {
     }
 
     const { args, ctx } = this.getRunArgs();
-    const after = runOnce(async () => {
-      if (this.hooks.after.length > 0) {
-        await this.runHook('after', { args, ctx });
-      }
-    });
+
     const afterEach = runOnce(async () => {
       if (this.parent?.hooks.afterEach.length > 0) {
         await this.parent.runHook('afterEach', { args, ctx });
       }
     });
+    const after = runOnce(async () => {
```

Comment:
Please undo this. It doesn't change anything and makes the diff bigger than it needs to be.

Reply:
Apologies, you're right. I only realised after I'd committed that I'd moved this for no reason. It's all back to where it was with a clean diff of the actual changes now. Thanks for looking over this.